templates/drone/index.ts

```diff
@@ -0,0 +1,106 @@
+import { 
```

Comment:
What if we add a checkbox (boolean property) to create the runner as well? Some people will deploy on the same server both the server and the runner, it could be useful to configure everything for them.

Reply:
Too easy, ill add the option.